compiler/il/AliasSetInterface.hpp

```diff
@@ -166,15 +166,15 @@ class TR_AliasSetInterface {
       {
       TR::Compilation *comp = TR::comp();
       LexicalTimer t("removeAliases", comp->phaseTimer());
-      static_cast<AliasSetInterface*>(this)->setAliases_impl(aliasesToModify, false,  _isDirectCall, _includeGCSafePoint);
+      (this)->setAliases_impl(aliasesToModify, false,  _isDirectCall, _includeGCSafePoint);
```

Comment:
Is it possible to fix the typo in the commit message (for setAlisa_impl) ?

Reply:
Sorry for the typo, I have fix the problem in 2d16c73.